libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -69,7 +71,8 @@ private data class DynamicallyUpdatedRouteValues(
     var sessionArrivalTime: AtomicReference<Date?> = AtomicReference(null),
     var sdkId: String = "none",
     val sessionStarted: AtomicBoolean = AtomicBoolean(false),
-    val originalRoute: AtomicReference<DirectionsRoute?> = AtomicReference(null)
+    val originalRoute: AtomicReference<DirectionsRoute?> = AtomicReference(null),
+    val isFreeDrive: AtomicBoolean = AtomicBoolean(true) // Default is IDLE, which is treated same as FREE_DRIVE
```

Comment:
What about working around the `isActiveGuidance` use case instead? I think it'd be easier to understand because that's what we only care about in terms of telemetry.

Reply:
👍